wm/desk.go

```diff
@@ -18,15 +18,34 @@ import (
 
 type x11WM struct {
 	stack
-	x      *xgbutil.XUtil
-	root   fyne.Window
-	rootID xproto.Window
-	loaded bool
+	x                   *xgbutil.XUtil
+	root                fyne.Window
+	rootID              xproto.Window
+	loaded              bool
+	moveResizing        bool
+	moveResizingStartX  int16
```

Comment:
"Start" seems like a poor name as this is not atually the start value, more like the last or previous value.